### ISSUE TITLE
[Issue]: file line endings on unix

### ISSUE THREAD

**Issue by CapsAdmin** [vladmandic/automatic#957]
### Issue Description

I believe the only problematic file is webui.sh as it won't execute on macos (and linux?) out of box unless you convert it to LF first.

I was going to make a pull request for one file, but then I forgot that line ending issues are more config related. I believe you can just add a .gitattributes file with "* text=auto" and everything should work in theory.

### Version Platform Description

Version: 2ce17e72 Sun May 14 13:27:59 2023 -0400

**Comment by vladmandic** [comment]
Yeah, thats a legacy issue. I've already converted bunch of files, forgot about that one, will do it tomorrow.

**Comment by vladmandic** [comment]
done.
